pkg/cluster/cluster.go

```diff
@@ -295,6 +295,8 @@ func (c *Cluster) run(stopC <-chan struct{}) {
 				break
 			}
 
+			garbagecollection.CollectDeadMemberServices(c.cluster.Name, c.cluster.Namespace, c.config.KubeCli, c.members, c.logger)
```

Comment:
> good code isn't my goal here

This has to be our #1 goal from now on. Or the codebase will become unmaintainable.